### PR TITLE
Line plts vis

### PR DESCRIPTION
**Pull request by msleckman**
This PR produces a couple additional outputs: 
- In `p4_export_csv.R` all the nwis sw and gw dataframes with a date range between 2000-2020 are exported to csv or folder of csvs. 
- In `3_visualize.R` we produce folders of measurement over time line plots per sw (x2) and gw (x1) parameter for each lake.

### CODE REVIEW

**Review by elmeraa — APPROVED** [review]
👍

**Comment by elmeraa** [comment]
Looks good, got it to run

File: 1_fetch/in/SalineLakeBnds/SalineLakeBnds.prj [hunk 1]
```diff
@@ -0,0 +1 @@
+PROJCS["NAD_1983_Albers",GEOGCS["GCS_North_American_1983",DATUM["D_North_American_1983",SPHEROID["GRS_1980",6378137.0,298.257222101]],PRIMEM["Greenwich",0.0],UNIT["Degree",0.0174532925199433]],PROJECTION["Albers"],PARAMETER["False_Easting",0.0],PARAMETER["False_Northing",0.0],PARAMETER["Central_Meridian",-96.0],PARAMETER["Standard_Parallel_1",29.5],PARAMETER["Standard_Parallel_2",45.5],PARAMETER["Latitude_Of_Origin",23.0],UNIT["Meter",1.0]]
```
Comment (padilla410) [inline 1_fetch/in/SalineLakeBnds/SalineLakeBnds.prj:1]: I'm looking at the list of files committed in `1_fetch/in/SalineLakeBnds` and I don't see the shapefile (shp)

**Comment by jesse-ross** [comment]
> Lingering issue for @elmeraa and @padilla410 is that target `p1_lake_flowlines_huc8_sf` builds, however it cannot load.

I think I've seen this before when trying to `tar_load()` an object while `sf` wasn't loaded. It's probably not a problem for the downstream target because that's run in an environment which has `sf` loaded.

Try `library(sf)` and then run  the `tar_load()` again?

**Pull request merged**